### ISSUE TITLE
error: Build input file cannot be found: Model.mlmodel

### ISSUE THREAD

**Issue by glenn-jocher** [cloud-annotations/classification-ios#4]
Xcode returns the following error when I attempt to send the app to my iPhone straight after a git clone. Where can I download the accompanying `Model.mlmodel`? Thank you.

```swift
error: Build input file cannot be found: '/Users/glennjocher/XcodeProjects/classification-ios/Core ML Classification/model_ios/Model.mlmodel' (in target 'Core ML Vision' from project 'Core ML Classification')
```

<img width="1128" alt="Screen Shot 2020-02-28 at 2 42 56 PM" src="https://user-images.githubusercontent.com/26833433/75592971-a02a0c00-5a38-11ea-81aa-209220467725.png">

**Comment by bourdakos1** [comment]
Follow these instructions on how to train a model https://cloud.annotations.ai/workshops/classification/index.html

**Comment by glenn-jocher** [comment]
@bourdakos1 thanks! My confusion stemmed from the Model.mlmodel in the project directory but not in the git clone. I was able to slot in the FlowerShop model to test the app and it works well.

This is off-topic, but I noticed there is unmentioned code in the app for a clever heatmap showing the confidence gradient over the image space, which apparently should appear when one clicks a category in the classification results. 

In my case this resulted in a hung screen for several minutes, at which point I restarted the app:
![IMG_EFA47331A8D8-1](https://user-images.githubusercontent.com/26833433/75714798-33558280-5c81-11ea-9d3e-901a5bcf57b1.jpeg)
